<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="24.010" tests="1" failures="1">
  <testsuite name="Root Suite" timestamp="2025-09-11T02:08:29" tests="0" file="cypress\e2e\login.cy.ts" time="0.000" failures="0">
  </testsuite>
  <testsuite name="Login flow" timestamp="2025-09-11T02:08:29" tests="1" time="24.004" failures="1">
    <testcase name="Login flow redirects to login, authenticates and lands on dashboard" time="0.000" classname="redirects to login, authenticates and lands on dashboard">
      <failure message="Timed out retrying after 15000ms: expected &apos;/&apos; to include &apos;/login&apos;" type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 15000ms: expected '/' to include '/login'
    at Context.eval (webpack://shadcn/./cypress/e2e/login.cy.ts:16:50)]]></failure>
    </testcase>
  </testsuite>
</testsuites>